test/integration/targets/docker_secret/tasks/Fedora.yml

```diff
@@ -2,7 +2,7 @@
   dnf:
     name: "{{ item }}"
     state: present
-  items:
+  loop:
```

Comment:
Why change this? Why change this? Seems out of scope.

Reply:
these are examples of tests relying on the bug described above

Reply:
:+1: